test/built-ins/Temporal/Instant/prototype/round/negative-instant.js

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant.prototype.round
+description: RoundNumberToIncrementAsIfPositive should multiply the remainder by an extra sign
+             before comparing it
```

Comment:
To fix the linter error:
```suggestion
description: >
  RoundNumberToIncrementAsIfPositive should multiply the remainder by an extra sign
  before comparing it
```
(same below)